corehq/apps/accounting/static/accounting/js/pricing_table.js

```diff
@@ -101,13 +123,17 @@ hqDefine('accounting/js/pricing_table', function () {
             pricingTable = pricingTableModel(
                 initial_page_data('editions'),
                 initial_page_data('current_edition'),
-                initial_page_data('is_renewal')
+                initial_page_data('is_renewal'),
+                true
             );
 
         // Applying bindings is a bit weird here, because we need logic in the modal,
         // but the only HTML ancestor the modal shares with the pricing table is <body>.
         $('#pricing-table').koApplyBindings(pricingTable);
         $('#modal-downgrade').koApplyBindings(pricingTable);
+        $('#toggle-annual-pricing').koApplyBindings(pricingTable);
```

Comment:
Left a long comment on this in the 30 day minimum PR, but you probably want to be applying bindings here just once, to some container element, and then maybe a second time to `#modal-downgrade` (since modals are usually put at the very end of the HTML, there may not be a good container that encloses both the main page content and the modal).